src/dungeon_ai_attack.c

```diff
@@ -47,6 +56,27 @@ extern void sub_8057588(Entity *, u32);
 extern void sub_806A1B0(Entity *);
 extern Item *sub_8044D90(Entity *, s32, u32);
 extern bool8 sub_8044D40(ActionContainer *, u32);
+extern bool8 EntityExists(struct Entity *entity);
```

Comment:
A few of these functions should be in headers already.. Rather use those first before externing.